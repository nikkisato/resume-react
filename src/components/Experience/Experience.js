import React from 'react';
import data from '../resume2';

export default function Education() {
  return (
    <div>
      <h1>Experience</h1>
      <h2>{data.Experiences[0].companyName}</h2>
      <p>Title: {data.Experiences[0].roles[0].title}</p>
      <p>Start Date: {data.Experiences[0].roles[0].startDate}</p>
      <p>End Date: {data.Experiences[0].roles[0].endDate}</p>

      <h2>{data.Experiences[1].companyName}</h2>
      <p>Title: {data.Experiences[1].roles[0].title}</p>
      <p>Start Date: {data.Experiences[0].roles[0].startDate}</p>
      <p>End Date: {data.Experiences[0].roles[0].endDate}</p>
    </div>
  );
}
