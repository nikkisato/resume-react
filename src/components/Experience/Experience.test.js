import React from 'react';
import { shallow } from 'enzyme';
import Experience from './Experience';

describe('Experience component', () => {
  it('renders Experience', () => {
    const wrapper = shallow(<Experience />);
    expect(wrapper).toMatchSnapshot();
  });
});
