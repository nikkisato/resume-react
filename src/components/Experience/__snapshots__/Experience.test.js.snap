// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Experience component renders Experience 1`] = `
<div>
  <h1>
    Experience
  </h1>
  <h2>
    Phototainment
  </h2>
  <p>
    Title: 
    Photographer
  </p>
  <p>
    Start Date: 
    2018
  </p>
  <p>
    End Date: 
    Present
  </p>
  <h2>
    Portland State University
  </h2>
  <p>
    Title: 
    Helpdesk and Desktop Service Tech
  </p>
  <p>
    Start Date: 
    2018
  </p>
  <p>
    End Date: 
    Present
  </p>
</div>
`;
